server/src/internalClusterTest/java/org/opensearch/search/pit/PitMultiNodeIT.java

```diff
@@ -246,7 +246,7 @@ public void testConcurrentCreates() throws InterruptedException {
         AtomicInteger numSuccess = new AtomicInteger();
         TestThreadPool testThreadPool = null;
         try {
-            testThreadPool = new TestThreadPool(DeletePitMultiNodeTests.class.getName());
+            testThreadPool = new TestThreadPool(PitMultiNodeIT.class.getName());
```

Comment:
why did we change Class name from `DeletePitMultiNodeIT` to `PitMultiNodeIT` here?

Reply:
the class name is incorrect before, so corrected it.